Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Menu/Menu.bind

```diff
@@ -24,112 +138,84 @@
-        </MenuFlyoutSubItem>
         <MenuFlyoutItem controls:Menu.InputGestureText="Ctrl+Shift+E"
                         Command="{StaticResource GenericCommand}"
                         CommandParameter="Solution closed"
-                        Text="Exit" />
+                        Text="Exit"
+                        Margin="0,0,0,8" />
       </controls:MenuItem>
 
-
-      <controls:MenuItem controls:Menu.InputGestureText="Alt+E"
-                         Header="^Edit">
-        <MenuFlyoutSubItem Text="Go To">
-          <MenuFlyoutItem Text="Go To Line" />
-          <MenuFlyoutItem Text="Go To All" />
-        </MenuFlyoutSubItem>
-
-        <MenuFlyoutSubItem Text="Find and Replace">
-          <MenuFlyoutItem Text="Find" />
-          <MenuFlyoutItem Text="Replace" />
-        </MenuFlyoutSubItem>
+      <controls:MenuItem controls:Menu.InputGestureText="Alt+E" Header="^Edit">
+        <MenuFlyoutItem controls:Menu.InputGestureText="Ctrl+X"
+                        Command="{StaticResource GenericCommand}"
+                        CommandParameter="Cut"
+                        Text="Cut"
+                        Icon="Cut" />
+        <MenuFlyoutItem controls:Menu.InputGestureText="Ctrl+C"
+                        Command="{StaticResource GenericCommand}"
+                        CommandParameter="Copied"
+                        Text="Copy"
+                        Icon="Copy" />
+        <MenuFlyoutItem controls:Menu.InputGestureText="Ctrl+V"
+                        Command="{StaticResource GenericCommand}"
+                        CommandParameter="Pasted"
+                        Text="Paste"
+                        Icon="Paste" />
       </controls:MenuItem>
 
       <controls:MenuItem controls:Menu.InputGestureText="Alt+V"
                          Header="^View">
-        <MenuFlyoutSubItem Text="Designer">
-          <MenuFlyoutItem Text="VS Designer" />
-          <MenuFlyoutItem Text="Blend" />
-        </MenuFlyoutSubItem>
-
-        <MenuFlyoutSubItem Text="Solution Explorer">
-          <MenuFlyoutItem Text="Solutions" />
-          <MenuFlyoutItem Text="Projects" />
-        </MenuFlyoutSubItem>
-      </controls:MenuItem>
-
-      <controls:MenuItem controls:Menu.InputGestureText="Alt+P"
-                         Header="^Project">
-        <MenuFlyoutSubItem Text="Add Class">
-          <MenuFlyoutItem Text="Class" />
-          <MenuFlyoutItem Text="Interface" />
-        </MenuFlyoutSubItem>
-
-        <MenuFlyoutSubItem Text="Add new data source">
-          <MenuFlyoutItem Text="SQL" />
-          <MenuFlyoutItem Text="NoSQL" />
-        </MenuFlyoutSubItem>
-      </controls:MenuItem>
-
-      <controls:MenuItem controls:Menu.InputGestureText="Alt+B"
-                         Header="^Build">
-        <MenuFlyoutItem Text="Build Solution" />
-        <MenuFlyoutItem Text="Rebuild Solution" />
+        <interactivity:Interaction.Behaviors>
```

Comment:
I like having the single item without a dropdown in the sample to show how to do this behavior.  However, I can't activate it with the keyboard.  So I think we also need to hook into the KeyDown event for the Enter key?